egs/hub4_english/s5/run.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+
+# Copyright 2016   Vimal Manohar
+# Apache 2.0.
+
+# See README.txt for more info on data required.
+
+. cmd.sh
+. path.sh
+
+set -o pipefail
+
+mfccdir=`pwd`/mfcc
+nj=40
+
+# 1996 English Broadcast News Train (HUB4)
+HUB4_96_Train_Transcripts=/export/corpora/LDC/LDC97T22/hub4_eng_train_trans
```

Comment:
I don't like the mixed-case variable names-- I'd prefer these to be all lowercase.